makefile

```diff
@@ -0,0 +1,34 @@
+# load and export envfile
```

Comment:
Where can we find the instruction to use the makefile?

Reply:
I have updated the `How can this be manually tested` section of the PR description to include this. After merging the PR, the readme should be updated to contain the instructions.